velox/functions/lib/DateTimeFormatter.h

```diff
@@ -206,7 +206,8 @@ class DateTimeFormatter {
       const tz::TimeZone* timezone,
       const uint32_t maxResultSize,
       char* result,
-      bool allowOverflow = false) const;
+      bool allowOverflow = false,
+      const std::optional<std::string>& zeroOffsetText = std::nullopt) const;
```

Comment:
nit: can you please update the method comment to add context about this input param.